convert2rhel/toolopts.py

```diff
@@ -211,6 +209,15 @@ def _register_options(self):
             " not provided, the subscription-manager defaults will be"
             " used.",
         )
+        group.add_option(
+            "--keep-rhsm",
+            action="store_true",
+            help="Do not remove the already installed Red Hat Subscription Management-related packages. By default,"
```

Comment:
Code LGTM, I would change this though as `--keep-rhsm` has a positive connotation while help message starts with a negative one. Comparatively, `--disable-submgr` has negative connotation on both the command and help message.

```suggestion
            help="Keep the already installed Red Hat Subscription Management-related packages. By default,"
```